### ISSUE TITLE
Font Opacity on 2.0 does not work in the same way?

### ISSUE THREAD

**Issue by iamthemovie** [zeit/hyper#2827]
- [X] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [X] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10 x64 (Redstone 3)
- **Hyper.app version**: 2.0
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/iamthemovie/71c924d36400df5d582f93adbf87d104
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: 
- **The issue is reproducible in vanilla Hyper.app**: Using the `hyper-material-box` and `hyperlinks`.

## Issue

Just updated to 2.0 and it looks like the font opacity is stuck. I was running the latest of 1.X hyper and this was working fine but literally just updated and my font is no longer the correct opacity. I realise this plugin maybe at fault but since it worked on 1.X and now has issues on 2.X I wondered if anyone could point in the direction where this problem might stem from?

1.0 (Nice and lovely)
![image](https://user-images.githubusercontent.com/369020/38822482-87498682-419b-11e8-8daa-b981b4e2ff9a.png)

2.0 (Still nice, not as easy to love)
![image](https://user-images.githubusercontent.com/369020/38822515-9a358156-419b-11e8-810b-b3cd5c0b4d98.png)
![image](https://user-images.githubusercontent.com/369020/38822340-0fcaa3fc-419b-11e8-9ce0-963137f97f96.png)

**Comment by chabou** [comment]
Yes default colors has been changed: https://github.com/zeit/hyper/pull/2670
We are now using iTerm2 defautls and it permits usage of bright colors.

You can restore previous color scheme by setting in your config, old color scheme:
```
    colors: {
      black: '#000000',
      red: '#ff0000',
      green: '#33ff00',
      yellow: '#ffff00',
      blue: '#0066ff',
      magenta: '#cc00ff',
      cyan: '#00ffff',
      white: '#d0d0d0',
      lightBlack: '#808080',
      lightRed: '#ff0000',
      lightGreen: '#33ff00',
      lightYellow: '#ffff00',
      lightBlue: '#0066ff',
      lightMagenta: '#cc00ff',
      lightCyan: '#00ffff',
      lightWhite: '#ffffff'
  }
```

**Comment by Astrophizz** [comment]
Even themes look dimmer than they did in 1.x. It's not just a matter of the default color scheme changing.

**Comment by iamthemovie** [comment]
Totally agree, this didn’t fix anything for me.

**Comment by mashaal** [comment]
I am experiencing the same issue -- it seems the culprit is a line of css is setting my active window to `.5` opacity. I've disabled all of my plugins and issue still persists.

```
#hyper .term_fit:not(.term_term) {
  opacity: .5
}
```

![hyper](https://user-images.githubusercontent.com/6100364/38915974-fa03c442-4328-11e8-8b88-3c9644383865.gif)

**Comment by iamthemovie** [comment]
@mashaal does this that take into account active pane? For instance if you split then before 2.0 the opacity would be 0.6 on the non-active window (which is super useful). I know it's a temporary fix, I'm just putting this comment for the guys at zeit's benefit 😄

**Comment by mashaal** [comment]
@iamthemovie i saw on a higher up node there was an active class with an opacity set -- not totally sure what the `term_fit` opacity is meant to be controlling. I feel like this may be an issue with conflicting css -- please see gif below -- in the inspector it shows the css line being overridden, but the `.5` opacity is still being rendered regardless?

![hyper-conflict](https://user-images.githubusercontent.com/6100364/38963079-9077c588-43b3-11e8-87b3-a587965c538d.gif)

**Comment by danielcorin** [comment]
Admittedly a hacky fix as I haven't done my due diligence on the use of these class selectors in the css but the following works for me on Hyper 2.0.0 (stable):

```
css: `
  .terminal:not(.focus) {
    opacity: 0.6;
  }
  .terminal.focus {
    opacity: 1;
  }`,

```

**Comment by pvorona** [comment]
For some reason in my hyper this css is in style tag (I've changed 0.6 to 1 but still WTF? why is it here?):
<img width="642" alt="screen shot 2018-04-26 at 21 51 37" src="https://user-images.githubusercontent.com/11915087/39325940-362cad98-499c-11e8-9c1c-e1fd65addc9d.png">

**Comment by chabou** [comment]
I added inactive pane dimming in `hyper-pane` plugin recently.
CSS class selection should be done carefully. Please see:
https://github.com/chabou/hyper-pane/commit/75aba86318259c8df00d9ba0713b4e2de9ba9a80

**Comment by ahmadawais** [comment]
I'm using this to debug this issue.

```js
// Custom css to embed in the main window.
    css: `
      .term_fit:not(.term_term) {
        opacity: 1 !important
      }
      .terminal: not(.focus) {
        opacity: 1;
        }
      .terminal.focus {
        opacity: 1;
      }
    `,
```
